salt/runners/git_pillar.py

```diff
@@ -104,4 +104,5 @@ def update(branch=None, repo=None):
         else:
             raise SaltRunnerError("No git_pillar remotes are configured")
 
+    print(f"DGM runners git_pillar update, exit ret '{ret}'", flush=True)
```

Comment:
Probably don't want that

Reply:
dang habit, thought I got them all, did a grep -R DGM, rats, it is gone